manifests/plugins/common.pp

```diff
@@ -0,0 +1,15 @@
+define etherpad::plugins::common (
+  $plugin_name = $title,
+  ) {
+    nodejs::npm { $plugin_name :
```

Comment:
You need to `include nodejs` before using `nodejs::npm`.

Reply:
Fixed.